Content.Server/Wieldable/WieldableSystem.cs

```diff
@@ -75,7 +75,7 @@ public bool CanWield(EntityUid uid, WieldableComponent component, EntityUid user
                 return false;
             }
 
-            if (hands.GetFreeHands() < component.FreeHandsRequired)
+            if (hands.Hands.Values.Count(hand => hand.IsEmpty) < component.FreeHandsRequired)
```

Comment:
would be nice if there was still a helper for this

Reply:
I added `HandHelpers.cs`